### PR TITLE
Common sync schema collection for all tests

### PR DESCRIPTION
**Pull request by desistefanova**
This PR is created to avoid removing schemas when breaking changes are allowed in dev mode in the new baas version.

### CODE REVIEW

**Review by nielsenko — APPROVED** [review]
LGTM

**Comment by nielsenko** [comment]
.. once all the test pass 😅

File: test/test.dart [hunk 1]
```diff
@@ -806,3 +807,18 @@ void printSplunkLogLink(AppNames appName, String? uriVariable) {
       "https://splunk.corp.mongodb.com/en-US/app/search/search?q=search index=baas$host \"${app.uniqueName}-*\" | reverse | top error msg&earliest=-7d&latest=now&display.general.type=visualizations");
   print("Splunk logs: $splunk");
 }
+
+List<SchemaObject> getSyncSchema() {
```
Comment (blagoev) [inline test/test.dart:811]: Perhaps we can add a comment why we are doing this

**Review by blagoev — APPROVED** [review]

File: test/test.dart (hunk 1 above)
Comment (desistefanova) [reply test/test.dart:816]: A comment is added.

**Pull request merged**